src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -61,10 +88,11 @@
      */
     public static final int EMAIL_MAX_LENGTH = 254;
     public static final String EMAIL_ERROR_MESSAGE = 
-            "\"%s\" is not acceptable to TEAMMATES as an email because it %s. "+
-            "An email address contains some text followed by one '@' sign followed by some more text. " +
-            "It cannot be longer than "+EMAIL_MAX_LENGTH+" characters. " +
-            "It cannot be empty and it cannot have spaces.";
+            "<b>Email</b> must contain some text followed by one '@' sign followed by some more text. "
+            + "It cannot be longer than "+ EMAIL_MAX_LENGTH +" characters and cannot have spaces.";
+    public static final String EMAIL_ERROR_MESSAGE_WITHOUT_HTML =
+            "Email must contain some text followed by one '@' sign followed by some more text. "
+            + "It cannot be longer than "+ EMAIL_MAX_LENGTH +" characters and cannot have spaces.";
```

Comment:
A non-HTML version is needed for testing purposes because the `WebElement.getText()` used in tests strips away the `<b></b>` HTML tag